en/option/series/graph.md

```diff
@@ -213,6 +213,15 @@ The label style of node in this category.
 #### label(Object)
 {{ use:partial-label(prefix="####") }}
 
+## autoCurveness(number|Array) = 20
+
+For the situation where there are multiple links between nodes, the curveness of each links is automatically calculated.
+
+When set to `Number`, it indicates the length of the edge curveness array between two points, and the calculation result is given by the internal algorithm.
+
+When set to `Array`, it means that the curveness array is directly specified, and the multilateral curvature is directly selected from the array.
+
+**Notice：** if [lineStyle.curveness](~series-graph.lineStyle.curveness) has been setted, this property is invalid.

```

Comment:
I'm afraid there is no such usage of `setted`, the past participle of `set` is still `set`.